speech/grpc/transcribe_async.py

```diff
@@ -85,9 +85,14 @@ def main(input_uri, encoding, sample_rate, language_code='en-US'):
         print('Waiting for server processing...')
         time.sleep(1)
         operation = service.GetOperation(
-            operations_grpc_pb2.GetOperationRequest(name=name),
+            operations_pb2.GetOperationRequest(name=name),
             DEADLINE_SECS)
 
+        if operation.error.message:
+            print("")
+            print("Operation error:")
+            print(operation.error)
```

Comment:
What? Jon didn't wasn't like, "Please use single quotes for strings"?? WHO ARE YOU AND WHAT HAVE YOU DONE TO @jonparrott ?!? :)

I'd just combine this into one string, for brevity:

    print('\nOperation error:\n{}'.format(operation.error))

We should be paid proportional to like, the inverse of lines of code :) Except then we'd be incentivized to not write any code at all... hm...